R/plot.ebp.R

```diff
@@ -30,14 +30,12 @@ plot.ebp <- function(x,
     )
     if (is.null(cooksdist)) {
       cooks <- FALSE
-      warning(
-        paste0(
-          "Cook's distance could not be calculated, this is usually due",
-          " to exceedence of available memory. Try using
-               cooks = FALSE to ",
-          "avoid this message and improve computation time."
-        )
-      )
+      warning(strwrap(prefix = " ", initial = "",
+                      paste0("Cook's distance could not be calculated, this is
+                             usually due to exceedence of available memory. Try
+                             using cooks = FALSE to avoid this message and
+                             improve computation time."
+                             )))
```

Comment:
```suggestion
                      "Cook's distance could not be calculated, this is
                      usually due to exceedance of available memory. Try
                      using cooks = FALSE to avoid this message and
                      improve computation time."
                      ))
```